### PR TITLE
CPS asset page headers

### LINKED ISSUE #4179: MAP - Add Metadata
**Is your feature request related to a problem? Please describe.**
We need to add the metadata to the MAP / CPS asset page

**Describe the solution you'd like**
Similar to what the other page types do

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/4179

**Overall change:** Add metadata and linked data to MAP

**Code changes:**

- Add metadata and linked data components to CPS asset container

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ibMadbouly — APPROVED** [review]
LGTM 😎 👍

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
-const CpsAssetPageMain = () => {
   return (
-    <Grid as="main" role="main">
-      <GridItemConstrainedMedium>
-        <h1> Placeholder content for MAP page skeleton</h1>
-      </GridItemConstrainedMedium>
-    </Grid>
+    <>
+      <MetadataContainer
+        title={promo.headlines.headline}
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/index.jsx]: As `promo.headlines.headline` is 3 levels deep could we use `pathOr` from `ramda` for safety

Comment (pjlee11) [reply src/app/containers/CpsAssetPageMain/index.jsx]: Same for `metadata.language` and `promo.summary`

File: src/app/containers/CpsAssetPageMain/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -89,16 +89,225 @@ exports[`CpsAssetPageMain should match snapshot 1`] = `
...
-       Placeholder content for MAP page skeleton
-    </h1>
-  </div>
-</main>
+  <head>
+    <title>
+      Nigerian man Emeka Nelson don produce generator wey dey use only water - BBC News Pidgin
+    </title>
+    <link
+      href="https://www.test.bbc.com/pathname"
```
Comment (pjlee11) [inline src/app/containers/CpsAssetPageMain/__snapshots__/index.test.jsx.snap]: Might be nice to update `pathname` to be a URL matching the Pidgin data

**Review by pjlee11 — CHANGES_REQUESTED** [review]

File: src/app/containers/CpsAssetPageMain/index.jsx [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
-      </GridItemConstrainedMedium>
-    </Grid>
+    <>
+      <MetadataContainer
+        title={title}
+        lang={metadata.language}
+        description={summary}
+        openGraphType="website"
+      />
+      <LinkedData type="Article" seoTitle={title} />
```
Comment (chris-hinds) [inline src/app/containers/CpsAssetPageMain/index.jsx:22]: is this container only going to be used for Articles? Could the hardcoding of `article` become a problem here

Comment (ryanmccombe) [reply src/app/containers/CpsAssetPageMain/index.jsx:22]: I believe Article is the correct type for MAP and STY pages - not sure what other page types are going to be rendered by this

I do have some doubt about "Article" being correct for MAPs - it seems correct technically as MAPs are just articles with a video at the top, but there seems to be a sentiment that editorial don't actually care about the article content below the media - so Article might not be correct

cc @aeroplanejane

Comment (chris-hinds) [reply src/app/containers/CpsAssetPageMain/index.jsx:22]: Ok thanks, I was thinking about the new feature indexes (FIX assets) however at the moment they are using the FrontPage container

Comment (ryanmccombe) [reply src/app/containers/CpsAssetPageMain/index.jsx:22]: Just had a meeting with Tom Chandler on this, and there's a lot of work needing analysed and implemented for this - different schemas per page, and per media type (audio MAPs have different schemas from video MAPs)

I'd recommend going with this for now (it's better than what we currently have - nothing) and creating an issue to track the v2 improvements

**Review by HarveyPeachey — APPROVED** [review]
LGTM 👍

**Review by pharingee — APPROVED** [review]
LGTM

**Review by pjlee11 — APPROVED** [review]
👍

**Review by dr3 — APPROVED** [review]

**Pull request merged**